lib/Mojolicious/Plugin/resources/public/css/responsive.css

```diff
@@ -0,0 +1,128 @@
+@media (max-width: 900px) {
+  pre {
+    max-width: 100%;
+    overflow: auto;
+  }
+
+  ul {
+    padding-left: 1.4em;
+  }
+
+  #perldoc,
+  .box {
+    padding-right: 1rem;
+    padding-left: 1rem;
+  }
+
+  #perldoc,
+  .spaced {
+    margin-left: 0;
+    margin-right: 0;
+    border: 0;
+  }
+
+  #fun {
+    background-position: 100% 50%;
+    background-size: contain;
+    height: 200px;
+    overflow: hidden;
+  }
+
+  #fun a {
+    display: block;
+  }
+
+  #fun img {
+    max-height: 240px;
+  }
+
+  #balloon {
+    width: 70px;
+    top: 10px;
+    right: 5px;
+  }
+
+  #mojobar {
+    overflow: visible !important;
```

Comment:
All of this "!important" stuff rules will go away, once I move the styling into mojo.